src/main/java/jenkins/plugins/slack/StandardSlackService.java

```diff
@@ -97,11 +105,12 @@ public StandardSlackService(StandardSlackServiceBuilder standardSlackServiceBuil
         if (standardSlackServiceBuilder.roomId == null) {
             throw new IllegalArgumentException("Project Channel or Slack User ID must be specified.");
         }
-        this.roomIds = standardSlackServiceBuilder.roomId.split("[,; ]+");
+        this.roomIds = new ArrayList<>(Arrays.asList(standardSlackServiceBuilder.roomId.split("[,; ]+")));
```

Comment:
@timja re: your question about only notifying the author of a PR. It seems that as long as the channel field is set to an empty string, then `roomIds ` will be initialized as an empty list. Only the committer's slack userId would be added to the list of roomIds if the `notifyCommitters` option was enabled.